tests/terraform/runner/test_plan_runner.py

```diff
@@ -27,10 +28,10 @@
    {"db_connector": IgraphConnector}
 ])
 class TestRunnerValid(unittest.TestCase):
-
-    def setUp(self) -> None:
-        self.orig_checks = resource_registry.checks
-        self.db_connector = self.db_connector
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.orig_checks = deepcopy(resource_registry.checks)
```

Comment:
why do you need to change this setup and teardown? seems like you are not using it anywhere, unless I missed something

Reply:
I guess that's a question for you @gruebel as you changes this part

Reply:
since this is a unittest testcase, the `setup` and `teardown` methods are invoked automatically before and after each test case. `setUpClass` is only invoked once for the whole file.